site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -96,22 +96,21 @@
 		            <input type="text" id= "user_id" value ="" placeholder="{{ user_id }}"/>
                 </div>
             </div>
-            <div class="sub hide" id="pdf-submit-button">
+            <div id="pdf-submit-button" class="sub hide">
```

Comment:
This change isn't necessary and should be reverted.

Reply:
This change was to have consistency with the rest of class= hide applied below.